src/components/shared/forms/pillContainer.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Pill } from './pill';
+import { Link } from 'utilities';
+import './styles/pillContainer.css';
+
+export const PillContainer = (props) => {
+  const { pills, onSvgClick, svg } = props;
```

Comment:
Same comment about doing the destructuring assignment in the function itself.

Reply:
Changed.